crates/router/src/types/connector_transformers.rs

```diff
@@ -0,0 +1,188 @@
+                })?
+            }
+            api_enums::Connector::Klarna => Self::Klarna,
+            api_enums::Connector::Mifinity => Self::Mifinity,
+            api_enums::Connector::Mollie => Self::Mollie,
+            api_enums::Connector::Moneris => Self::Moneris,
+            api_enums::Connector::Multisafepay => Self::Multisafepay,
+            api_enums::Connector::Netcetera => {
+                Err(common_utils::errors::ValidationError::InvalidValue {
+                    message: "netcetera is not a routable connector".to_string(),
+                })?
+            }
+            api_enums::Connector::Nexinets => Self::Nexinets,
+            api_enums::Connector::Nexixpay => Self::Nexixpay,
+            api_enums::Connector::Nmi => Self::Nmi,
+            api_enums::Connector::Nomupay => Self::Nomupay,
+            api_enums::Connector::Noon => Self::Noon,
+            // api_enums::Connector::Nordea => Self::Nordea,
+            api_enums::Connector::Novalnet => Self::Novalnet,
+            api_enums::Connector::Nuvei => Self::Nuvei,
+            api_enums::Connector::Opennode => Self::Opennode,
+            api_enums::Connector::Paybox => Self::Paybox,
+            api_enums::Connector::Payload => Self::Payload,
+            api_enums::Connector::Payme => Self::Payme,
+            api_enums::Connector::Payone => Self::Payone,
+            api_enums::Connector::Paypal => Self::Paypal,
+            api_enums::Connector::Paystack => Self::Paystack,
+            api_enums::Connector::Payu => Self::Payu,
+            api_models::enums::Connector::Placetopay => Self::Placetopay,
+            api_enums::Connector::Plaid => Self::Plaid,
+            api_enums::Connector::Powertranz => Self::Powertranz,
+            api_enums::Connector::Prophetpay => Self::Prophetpay,
+            api_enums::Connector::Rapyd => Self::Rapyd,
+            api_enums::Connector::Razorpay => Self::Razorpay,
+            api_enums::Connector::Recurly => Self::Recurly,
+            api_enums::Connector::Redsys => Self::Redsys,
+            api_enums::Connector::Santander => Self::Santander,
+            api_enums::Connector::Shift4 => Self::Shift4,
+            api_enums::Connector::Signifyd => {
+                Err(common_utils::errors::ValidationError::InvalidValue {
+                    message: "signifyd is not a routable connector".to_string(),
+                })?
+            }
+            api_enums::Connector::Riskified => {
+                Err(common_utils::errors::ValidationError::InvalidValue {
+                    message: "riskified is not a routable connector".to_string(),
+                })?
+            }
+            api_enums::Connector::Square => Self::Square,
+            api_enums::Connector::Stax => Self::Stax,
+            api_enums::Connector::Stripe => Self::Stripe,
+            api_enums::Connector::Stripebilling => Self::Stripebilling,
+            // api_enums::Connector::Taxjar => Self::Taxjar,
```

Comment:
Nit: I believe we can remove this commented code, considering that we're handling this variant later in the code.